packages/kit-styled/src/components/progress/progress.tsx

```diff
@@ -0,0 +1,16 @@
+import { cn } from '@qwik-ui/utils';
+import * as ProgressPrimitive from '@qwik-ui/headless';
```

Comment:
So this syntax of the <Component.Part /> has been brought up several times, as well as pretty recently.

While other headless libraries do this convention, we are not yet sure if it's a wise move. This is something we'll need to have a discussion on in the next Qwik UI meeting. 

The concern has to do with tree shaking. For example, in the carousel you may have a CarouselNext or CarouselPrevious button to go to the next or previous slide. If it's not being tree shaken properly, it's going to execute all of that code, even if you don't need it.

It could also potentially be a breaking change for consumers. For now, I would use the same convention as something like the select component.

Reply:
Makes sense, done.